### DISCUSSION TITLE
[Statistic Question] Minimum sample size for data analysis

### DISCUSSION THREAD

**Discussion by kevinyoman** [teamalgoritma/community#361] [Q&A] [answered]
Hi,

Sorry kalau out of topic, apakah ada 'universal rule' untuk jumlah minimum sample supaya dianggap relevan ? 
atau case-by-case ? 

kalau saya googling, jawabannya berbeda mulai dari kisaran 30 sampai 100.

Terima kasih

**Comment by onlyphantom** [comment, marked as answer]
Hey Kevin,

The reason you should be skeptical of any hard figure like 30 or 100 is because they fail to take into account the whole point of having a sample, which is to "get a representation" that could serve as an estimation of the actual population's statistic. Here's a simple example to do some brainstorming together.

1. Supposed you pick a sample size of 5 to estimate length of men's hand length. You got the following values: 20cm, 18cm, 19cm, 17cm and 21cm. 
2. Supposed you pick a sample size of 5 to estimate the average men's driving speed on the Gatot Subroto highway: 80km/h, 56km/h, 110km/h, 49km/h, 108km/h

So 5 is a very extreme example, pick only to demonstrate the point here so I don't have to present 30 values in the example. 

Which, (1) or (2) would you feel more confident about when trying to estimate for a range of the true population statistic? If you take the numbers up to 30, instead of sample size of 5, for case number (2), but the variance in the data are equally wide, it means even at 30 you are still not more confident, statistically speaking, and no closer to being able to estimate the population statistic. 

So the whole point about getting a large enough sample size is in reducing this variance. Men's finger length has lesser variance than stock performance over 30 years, so you need more sample for the latter and less for the former. Think of managing sample size as managing variance / standard deviation. The narrower the std deviation, the less sample size required to get to a level of confidence (an oversimplified explanation, but hope it helps think of the problem a bit differently :)

**Comment by kevinyoman** [comment]
Hi Samuel Chan,

Appreciate for quick reply, thank you for the explanation
